### ISSUE TITLE
Critical error. Rainbow crash -》not mining

### ISSUE THREAD

**Issue by ebojali** [RainbowMiner/RainbowMiner#75]
Seee picture

**Comment by RainbowMiner** [comment]
Is there any editor open, with the miners.config.txt file? It looks like a write-lock problem.
Anyway, I will take care of those write-locks asap.

RBM

**Comment by ebojali** [comment]
No. Three is none but I had it opened a  couple of days agoo. how can it be fixed manually? I restarded the pc but no luck there..

**Comment by RainbowMiner** [comment]
It looks like the miners.config.txt might be broken. You should try:
- open miners.config.txt with notepad
- save as with another name
- delete miners.config.txt
- copy your save as backup to miners.config.txt

RBM

**Comment by ebojali** [comment]
It worked now but the miner is not started at all anymore.. X17 but no powershellwindow with miner..![image](https://user-images.githubusercontent.com/35561346/44948238-5c8e1c00-ae1a-11e8-83d3-d1a5fb608302.png)

**Comment by RainbowMiner** [comment]
First, I will add a config backup to the next release.
Second, this is normal - I currently disabled the automatic start of the mining-window, because some user had problems with it (it will be re-enabled as soon as I have checked this).
 You can start it manually: click "MinerLog.bat".

**Comment by ebojali** [comment]
Alright that did the trick. Thanks
